prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -246,6 +246,46 @@
 
 </app-page-section>
 
+<app-page-section>
+
+  <app-page-subheader>
+    <ng-container appPageSubheaderTitle>Other College Licence Information</ng-container>
+
+    <button *ngIf="showEditRedirect"
+            mat-icon-button
+            matTooltip="Edit College Licences"
+            (click)="onRoute(EnrolmentRoutes.REGULATORY)">
+      <mat-icon>edit</mat-icon>
+    </button>
+  </app-page-subheader>
+
+  <ng-container *ngIf="!enrolment?.unlistedCertifications?.length; else unlistedCerts">
+    No unlisted certifications entered.
```

Comment:
Can you please check with someone from the Business whether this text makes sense to them?

Reply:
@neophyte57 @bergomi02 received the text today from the business and updated the code.